common/changes/@itwin/grouping-mapping-widget/arnobm-groupingmapping-upgrade-to-4x_2023-06-01-17-17.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/grouping-mapping-widget",
+      "comment": "New version of @itwin/grouping-mapping-widget targetting itwinjs 4.x and itwinui 2.x!",
```

```suggestion
      "comment": "New version of @itwin/grouping-mapping-widget targeting iTwinJs 4.x and iTwinUI 2.x!",
```
Tbh, is there a reason to mention the widget itself? `Updated to iTwinJs 4.0 and iTwinUI 2.0`. Can probably suffice.

---

I don't think you need to mention the package name in comment since this changelog entry will only appear for that one package.

---

updated